frontend/integration-tests/tests/secrets.scenario.ts

```diff
@@ -118,12 +118,7 @@ describe('Interacting with the create secret forms', () => {
         testName,
         sshSourceSecretName,
         async () => {
-          await secretsView.authTypeDropdown.click().then(() =>
-            browser
-              .actions()
-              .sendKeys(Key.ARROW_UP, Key.ENTER)
-              .perform(),
-          );
+          await secretsView.authTypeDropdown.click().then(() => secretsView.authSSHOption.click());
```

Comment:
Don't mix promises with async/await. Use async/await consistently.

Reply:
Sure, will be fixed in next push.

Reply:
PTAL